core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -505,7 +505,7 @@ private static String toNormalizedIconUrl(String url) {
         icons.addIcon(new Icon("icon-setting icon-lg", "32x32/setting.gif", Icon.ICON_LARGE_STYLE));
         icons.addIcon(new Icon("icon-star-gold icon-lg", "32x32/star-gold.gif", Icon.ICON_LARGE_STYLE));
         icons.addIcon(new Icon("icon-star icon-lg", "32x32/star.gif", Icon.ICON_LARGE_STYLE));
-        icons.addIcon(new Icon("icon-user icon-lg", "32x32/user.gif", Icon.ICON_LARGE_STYLE));
+        icons.addIcon(new Icon("icon-user icon-lg", "32x32/user.png", Icon.ICON_LARGE_STYLE));
```

Comment:
What's with line 532?

Reply:
I don't know why gif and png are in here, historical I assume, when felix switched things to svgs he removed the duplicates, we need to do the same cleanup when we switch the rest to SVG

https://github.com/jenkinsci/jenkins/pull/5065/files#r587464783